config/spinnaker.yml

```diff
@@ -294,3 +294,18 @@ providers:
       name: my-docker-registry-account
       address: https://index.docker.io/
       repository: library/nginx
+
+  openstack:
+    # For more information on configuring Openstack, see
+    # TODO Added doc link when available
+
+    enabled: false
+    primaryCredentials:
+      name: my-openstack-account
+      authUrl:
+      username:
+      password:
+      projectName:
+      domainName:
+      regions:
```

Comment:
Not really. Openstack admins decide on project & regions. Domain is the only thing that has a reasonable default.